docs/NDRC_Create_Claim.js

```diff
@@ -0,0 +1,210 @@
+        submit();
+    }
+    if (currentPageIs("/reclaim")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/customs-duty-overpayment")) {
+        document.getElementById("ActualPaidAmount").value = "100.00";
+        document.getElementById("ShouldHavePaidAmount").value = "89.99";
+        submit();
+    }
+    if (currentPageIs("/repayment-summary")) {
+        submit();
+    }
+    if (currentPageIs("/evidenceSupportingDocs")) {
+        submit();
+    }
+    if (currentPageIs("/file-uploaded")) {
+        document.getElementById("uploadAnotherFile-2").checked = true;
+        submit();
+    }
+    if (currentPageIs("/do-you-have-an-eori-number")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/does-importer-have-eori-number")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/enter-your-eori-number")) {
+        document.getElementById("value").value = "GB123456789123000";
+        submit();
+    }
+    if (currentPageIs("/enter-importer-eori-number")) {
+        document.getElementById("value").value = "GB123456789123111";
+        submit();
+    }
+    if (currentPageIs("/is-importer-vat-registered")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/vat-registered")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/representative-importer-name")) {
+        document.getElementById("importerName").value = "ACME Importer Ltc";
+        submit();
+    }
+    if (currentPageIs("/representative-agent-name")) {
+        document.getElementById("declarantName").value = "Tim Tester";
+        document.getElementById("agentName").value = "Tester Import Agents";
+        submit();
+    }
+    if (currentPageIs("/enter-your-name")) {
+        document.getElementById("firstName").value = "Tim";
+        document.getElementById("lastName").value = "Tester";
+        submit();
+    }
+    if (currentPageIs("/goods-owner")) {
+        document.getElementById("value").checked = true;
+        submit();
+    }
+    if (currentPageIs("/your-business-address")) {
+        document.getElementById("PostalCode").value = "AA000AA";
+        submit();
+    }
+    if (currentPageIs("/select-importer-address")) {
+        document.getElementById("PostalCode").value = "AA000AA";
+        submit();
+    }
+    if (currentPageIs("/importerManualAddress")) {
+        document.getElementById("AddressLine1").value = "Unit 42";
+        document.getElementById("AddressLine2").value = "West Industrial Estate";
+        document.getElementById("City").value = "Walsall";
+        document.getElementById("PostalCode").value = "WS1 2AB";
+        document.getElementById("CountryCode").getElementsByTagName("option")[1].selected = "selected";
```

Comment:
@SteveSugden the plan was to implement a https://github.com/alphagov/govuk-country-and-territory-autocomplete on countrycode field. Would your code work if this was implemented? If you're happy to address this when/if the dropdown is changed, that's fine.

Reply:
Hi @lmugnai - autocomplete's are always a problem!

I only created the PR as a test of whether I could create a PR!    We have TamperMonkey scripts as an "aid" for testers and team members doing demo's, etc.   Although we do try to keep them up-to-date (e.g. when a form changes or a new page is added) the scripts come without any guarantees.   So if the country picker starts to fail,  we will have to manually select a value until it is fixed.

Reply:
Thanks for clarifying @SteveSugden :)